modules/nf-core/cellranger/mkfastq/tests/main.nf.test.snap

```diff
@@ -4,19 +4,273 @@
+                                            [
+                                                
+                                            ],
+                                            [
+                                                
+                                            ],
+                                            [
+                                                
+                                            ]
+                                        ],
+                                        [
+                                            [
+                                                
+                                            ],
+                                            [
+                                                
+                                            ],
+                                            [
+                                                
+                                            ]
+                                        ],
+                                        [
+                                            [
+                                                
+                                            ],
+                                            [
+                                                
+                                            ],
+                                            [
+                                                
+                                            ]
+                                        ]
+                                    ],
+                                    [
+                                        [
+                                            [
+                                                
+                                            ],
+                                            [
+                                                
+                                            ]
+                                        ]
+                                    ],
+                                    [
+                                        [
+                                            [
+                                                
+                                            ],
+                                            [
+                                                
+                                            ]
+                                        ],
+                                        [
+                                            [
+                                                
+                                            ],
+                                            [
+                                                
+                                            ]
+                                        ]
+                                    ]
+                                ]
+                            ]
+                        ]
+                    ]
+                ],
```

Comment:
Is this nested empty array expected?

Reply:
This nested array is created from the reports output channel, which contains only empty folders. I added the reports and stats output channels to replicate what was done in the bcl2fastq module (https://github.com/nf-core/modules/blob/master/modules/nf-core/bcl2fastq/main.nf), in case the user requires this output files.
This main.nf.test uses the default bcl test sample provided by 10x genomics, and both test csv samplesheets also provided by 10x genomics. However, it seems that for this particular samples, it doesn't produce any files inside the reports directory.

 I believe that for the module nf-test its best to leave this datasets as input, since they are the official testing datasets, but I can remove the reports output channel, if it is not needed.
 
@grst  How would you like me to proceed?

Reply:
@grst I looked into the issue of the nf-test not always returning the same results, it seems that this only happens with the stats folder, which doesn't return deterministic results. I updated the main.nf.test to not check for these files, and I also removed the check of the reports file since it was producing empty folders.

Reply:
That's fine with me! Thanks!